pkg/morph/client/client.go

```diff
@@ -419,6 +421,30 @@ func (c *Client) IsValidScript(script []byte, signers []transaction.Signer) (res
 	return result.State == vmstate.Halt.String(), nil
 }
 
+// AccountVote returns a key the provided account has voted with its NEO
+// tokens for. Nil key with no error is returned if the account has no NEO.
```

Comment:
`Nil key with no error is returned if the account has no NEO` or when account has some NEO and didn't voted for anyone.